demisto_sdk/commands/generate_unit_tests/test_case_builder.py

```diff
@@ -0,0 +1,338 @@
+import ast as ast_mod
+import json
+import logging
+from pathlib import Path
+from typing import Dict, List
+
+from ordered_set import OrderedSet
```

Comment:
trying to minimize dependencies, please consider using `dict.from_keys()`, then treating `dict.keys()` as an ordered set, see [here](https://www.programiz.com/python-programming/methods/dictionary/fromkeys)

Reply:
same comment here, sorry